pages/sdk/bako-standalone/vault.mdx

```diff
@@ -13,6 +13,26 @@ const params: IPayloadVault = {
 }
```

Comment:
add call to safe code before:
`const params: IPayloadVault = {
  name: 'My First Vault',
  description: 'This is my first example of a Vault',
  configurable: {
    SIGNATURES_COUNT: 1,
    SIGNERS: [Address.fromRandom().toString(), Address.fromRandom().toString()],
    network: BakoSafe.getProviders('CHAIN_URL'),
  },`